### PR TITLE
Add type support for {home} and {end}

### PR DESCRIPTION
**Pull request by curiosity26**
**What**: I've added support for typing the Home and End keys

**Why**: Per aria-role spec for listbox, Home and End are necessary inputs that need to be handled and tested. Not only that, it's common in textbox input, so why not, right? 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role

**How**: Since Home and End are navigation keys, I added to the existing `navigationKeys` function to support `Home` and `End` and ensured cursor functionality worked as intended.

**Checklist**:

- [X] Documentation
- [X] Tests
- [X] Typings
- [X] Ready to be merged

### CODE REVIEW

File: src/keys/navigation-key.js
Comment (luistak) [inline src/keys/navigation-key.js:11]: Following [keycode.info](https://keycode.info/)'s table, those keys are inverted is it right?
image

**Comment by curiosity26** [comment]
Omg. Dyslexia strikes again. Let me fix that. Good catch.

File: src/keys/navigation-key.js
Comment (curiosity26) [reply src/keys/navigation-key.js]: @luistak I've corrected the blunder.

**Review by ph-fritsche — APPROVED** [review]

**Review by luistak — APPROVED** [review]

**Comment by audiolion** [comment]
Who makes the call to merge? This has been hanging out in approved for a while.

**Comment by kentcdodds** [comment]
I'm no longer an active maintainer. Any maintainer is welcome to merge a PR if you feel like it's a good direction. If you run into issues, let me know.

**Comment by kentcdodds** [comment]
@ph-fritsche can you not merge this yourself? We should give you merge access if you don't already have it.

**Comment by kentcdodds** [comment]
@ph-fritsche can you not merge this yourself? We should give you merge access if you don't already have it.

**Comment by kentcdodds** [comment]
Just gave you merge access 👍

**Comment by nickmccurdy** [comment]
@all-contibutors add @curiousity26 for code

**Comment by nickmccurdy** [comment]
@all-contibutors add @curiosity26 for code

**Comment by nickmccurdy** [comment]
@all-contributors add @curiosity26 for code

**Pull request merged**
